pkg/controlplane/apiserver/config.go

```diff
@@ -159,6 +165,15 @@ func BuildGenericConfig(
 		genericConfig.FlowControl, lastErr = BuildPriorityAndFairness(s, clientgoExternalClient, versionedInformers)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.UnknownVersionInteroperabilityProxy) {
+		genericConfig.PeerEndpointLeaseReconciler, lastErr = createPeerEndpointLeaseReconciler(*genericConfig, storageFactory)
+		if lastErr != nil {
+			return
+		}
+		genericConfig.PeerProxy, _ = BuildPeerProxy(versionedInformers, genericConfig.StorageVersionManager, s.ProxyClientCertFile,
```

Comment:
don't ignore this error, avoid calling this if PeerCAFile is unspecified

Reply:
[Updated](https://github.com/kubernetes/kubernetes/pull/117740/files#diff-170c324cc9e649c56eba833146296ea83b4fe433cb1c05b63ca18578609da4d2R269-R273)